### ISSUE TITLE
Issues with make netstandard on Ubuntu 16.04

### ISSUE THREAD

**Issue by flashfm** [google/or-tools#635]
Hi,
I found 2 issues preventing `make netstandard` working on Ubuntu 16.04:
1. `tools/netstandard/CreateSigningKey/Program.cs` cannot be compiled
2. Makefile.csharp.mk line
```
@echo [assembly: AssemblyKeyFile("$(notdir $(CLR_KEYFILE))")] >> $(NETSTANDARD_OBJ_DIR)$SAssemblyInfo.cs 
```
is wrong because of quote char, I had to add additional quotes and escape the inner quotes.

It seems that pull request https://github.com/google/or-tools/pull/586 targets these issues in particular, as well as https://github.com/google/or-tools/issues/634

**Comment by Mizux** [comment]
#668 Will provide lot of change, should test and fix it once it is integrated.
I.e. we are moving to `dotnet` first instead of `mono`

**Comment by Mizux** [comment]
using dotnet core sdk 302 >= CreatSigningKey seems to work on all targeted platform (cf tools/docker/*)
There was issue on unix concerning the resolution of `CurrentWorkingDirectory()`
cf: issues dotnet/cli#9073, dotnet/cli#9319 and the fix dotnet/cli#9504 in 2.1.3xx
